include/sound/sof/dai.h

```diff
@@ -52,12 +52,23 @@
 #define SOF_DAI_FMT_INV_MASK		0x0f00
 #define SOF_DAI_FMT_CLOCK_PROVIDER_MASK	0xf000
```

Comment:
Typo: "hw_params i.ei before"

I also wonder if it would be clearer to talk about "two-step flow for stopping the DAI DMA". If you just look at the kernel driver side code, we have really no "pipeline stop" action at all. What we do is "STREAM_TRIG_STOP". FW implementation looks up the PCM component based on the stream-id, and then finds a pipeline object, but this is all hidden from kernel. So a "two-spec stop for pipeline stop" seems really hard to understand as we don't even have a pipeline stop.  As in SOF arch, the DMAs are driven by the components, "two-step stop for the DAI" seems most accurate.

Reply:
makes sense. I've modifed the commit message to make it applicable specifically for DAIs